src/hooks/useAnimatedRef.ts

```diff
@@ -68,33 +64,26 @@ const runAnimation = <E extends HTMLElement>(
     options: AnimationOptions,
     onAnimationEnd: (webAnimation: Animation | null) => void,
     errorMessage: (err: Error) => string
-) => {
+): void | (() => void) => {
     try {
         const webAnimation = element.animate(keyframes, options as KeyframeAnimationOptions);
 
         webAnimation.onfinish = () => {
-            try {
```

Comment:
Behavior change here in that if you're using a raw useAnimatedRef (not under ControlledAnimated), it won't commitStyles onAnimationEnd for you. That's done by default under ControlledAnimated, but is now configurable via the prop